src/apis/stockx/index.js

```diff
@@ -0,0 +1,205 @@
+
+import { jar } from 'request-promise-native';
+
+// APIs
+import ProductsApi from './products';
+import AsksApi from './asks';
+import BidsApi from './bids';
+import UserApi from './user';
+
+// Utils
+import { format, currencies } from '../../utils';
+
+export default class StockX {
+
+  /**
+   *
+   * @param {Object} options
+   * @param options.proxy - The proxy to use for future requests
+   * @param options.currency - The currency used when making requests
+   */
+  constructor(options = {}){
+    const { proxy = null, currency = currencies.USD } = options;
```

Comment:
Should we default to `USD`?

Reply:
If it's documented this would probably be the most common, `options` has the opportunity to override the default, so I see no issue.

Reply:
Yes